css/styles.css

```diff
@@ -321,3 +402,290 @@ textarea {
+  position: absolute;
+  visibility: hidden;
+  width: 353px;
+  height: 351px;
+  left: -86px;
+}
+
+.imggeo2 {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+.imggeo5 {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+.fixGap1 {
+  margin-bottom: -5px;
+}
+
+.imgGroup83 {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+.redSquare {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+.imageDots2 {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+@media screen and (min-width: 1680px) {
+  .decoration1 {
+    visibility: hidden;
+  }
+
+  .decoration2 {
+    visibility: hidden;
+  }
+
+  .decoration3 {
+    background-image: url(../resources/Thur.png);
+    background-repeat: no-repeat;
+    padding-bottom: 164px;
+    height: 877.09px;
+    visibility: visible;
+    width: 1403px;
+    margin-left: 677px;
+    position: relative;
+  }
+
+  .desktopflex {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .desktopnav {
+    visibility: visible;
+  }
+
+  .dflex2 {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: center;
+  }
+
+  .grid1 {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+  }
+
+  .header1 {
+    margin-top: -94px;
+  }
+
+  .rectangle1 {
+    margin-bottom: 0;
+  }
+
+  .rectangle2 {
+    margin-left: 0;
+    margin-bottom: 12px;
+  }
+
+  .weirddeco1 {
+    position: relative;
+    visibility: visible;
+  }
+
+  .weirddeco2 {
+    position: absolute;
+    left: 34px;
+    top: 113px;
+  }
+
+  .weirddeco3 {
+    position: absolute;
+    left: 207px;
+    top: 51px;
+  }
+
+  .dottedimg1 {
+    visibility: visible;
+    position: absolute;
+    height: 175px;
+    width: 175px;
+    top: 701px;
+    left: -174px;
+  }
+
+  .awesomedeco1 {
+    visibility: visible;
+  }
+
+  .awesomedeco2 {
+    visibility: visible;
+  }
+
+  .getstarted {
+    width: 100%;
+    margin-left: 0;
+  }
+
+  .projectsrotate {
+    font-size: 110px;
+    transform: rotate(-90deg);
+  }
+
+  .box3 {
+    height: 0;
+    width: 0;
+    visibility: hidden;
+  }
+
+  .cooldeco1 {
+    visibility: visible;
+  }
+
+  .box1 {
+    position: absolute;
+    left: 880px;
+    top: -51px;
+    width: 960px;
+  }
+
+  .box2 {
+    width: 100%;
+  }
+
+  .noOF {
+    width: 960px;
+  }
+
+  .yellowdot {
+    position: absolute;
+    top: 1305px;
+    visibility: visible;
+    left: 979px;
+    height: 88px;
+    width: 88px;
+  }
+
+  .imggeo5 {
+    position: absolute;
+    top: 1303px;
+    height: 125px;
+    width: 212px;
+    visibility: visible;
+  }
+
+  .rectangle21 {
+    position: absolute;
+    top: 671px;
+    left: 1380px;
+    width: 444px;
+    z-index: -1;
+  }
+
+  .imggeo2 {
+    position: absolute;
+    height: 350px;
+    width: 511px;
+    left: 867px;
+    top: 671px;
+    visibility: visible;
+  }
+
+  .floatingBox {
+    position: absolute;
+    left: 1324px;
+    top: 762px;
+  }
+
+  .aboutTopMargin {
```

Comment:
- [ ] Kindly update this section text to make it match Figma (font and font-size are not the same) 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/187050371-8511f82b-33ba-40d2-82d2-7e8ef5ea54e2.png)

## Figma design: 
![image](https://user-images.githubusercontent.com/50721479/187050380-d2f88cfe-06ef-4918-a823-3134ef31bcf9.png)